domain_director/tests/test_server.py

```diff
@@ -36,3 +36,7 @@ def test_correct_request(self):
         rv = self.app.post('/', data={'wifis': json.dumps(post_data)},
                            environ_base={'REMOTE_ADDR': '2001:67c:2ed8:6100:fc64:3ff:fecd:45dd'})
         self.assertEqual(rv.status_code, 200)
+
+    def test_admin_mesh(self):
+        rv = self.app.patch('/admin/mesh/999999/', data={})
+        self.assertEqual(rv.status_code, 404)
```

Comment:
Also test for the HTTP 400 case here https://github.com/freifunk-darmstadt/ffda-domain-director/pull/3/files#diff-e852383f0904571fb8352b555a1566c1R28

Reply:
did you link to the correct line? the references line does not seem to be part of my changes ;)

Reply:
Possibly, what I mean is the `abort(400)` responses.

Reply:
I added more tests for all the paths